README.md

```diff
@@ -1,27 +1,83 @@
 # Syntropy PubSub-JS
 
-`syntropy-pubsub-js` is a TypeScript library for the Syntropy DataMesh project that allows you to subscribe to existing data streams or publish new ones. This library is built on top of the NATS messaging system and provides a convenient way to integrate your TypeScript applications with the Syntropy DataMesh platform.
+Welcome to the documentation for the JavaScript SDK for the Data Availability Layer by SyntropyNet! This SDK allows seamless integration with our Data Availability Layer solution, enabling you to leverage real-time data streams in your JavaScript applications. With the JavaScript SDK, you can unlock the power of the Data Availability Layer and harness real-time insights for your data-driven projects.
 
-## Features
-
-- Subscribe to existing data streams
-- Publish new data streams
-- Support for JSON messages
-- Customizable connection options
+`[syntropy-pubsub-js](https://github.com/SyntropyNet/pubsub-js)` is a TypeScript library for the Syntropy Data Availability Layer project that allows you to subscribe to existing data streams or publish new ones. This library is built on top of the NATS messaging system and provides a convenient way to integrate your TypeScript applications with the Syntropy Data Availability Layer platform.
 
 ## Installation
 
-To install the library, use the following command:
+To use the JavaScript SDK for Data Availability Layer in your project, you can include the provided script in your HTML:
+
+```html
+<script src="path/to/pubsub-js.js"></script>
+```
+
+Alternatively, you can install it using npm:
 
-```bash
-npm install git@gitlab.com:syntropynet/amberdm/sdk/pubsub-js.git
+```shell
+npm install syntropynet-pubsub-js
 ```
 
+## Getting Started
+
+Before you begin using the JavaScript SDK, make sure you have the necessary credentials and access tokens from the Syntropy Developer Portalplatform. These credentials will allow you to connect to the Data Availability Layer and subscribe to or publish data streams.
+
 ## Usage
-Here is a simple example demonstrating how to subscribe to a data stream using seed from developer-portal:
 
-### The preferred method of authentication is using an access token from the developer portal.
-```typescript
+1. Import the SDK in your JavaScript code:
+
+```javascript
+import { Data Availability LayerClient } from 'syntropynet-pubsub-js';
```

Comment:
I guess it should be `DataAvailabilityLayerClient` instead of `Data Availability LayerClient`